KIC 2.0 - Add 2.0 CHANGELOG & Remove ClassLess Ingress Support (#1328)
Kong/kubernetes-ingress-controller: CHANGELOG.md

# Table of Contents

>  - [2.0.0](#200---WIP)

Reviewer: We normally log the individual pre-releases before folding all changes into a single 2.0.0 entry at the final release, to show the specific changes in subsequent pre-release versions.
```suggestion
 - [2.0.0-alpha.1](#200-alpha1---TBD)
```

 - [0.0.5](#005---20180602)
 - [0.0.4 and prior](#004-and-prior)

> ## [2.0.0] - WIP

Reviewer 2: Now we know that we'll mark the new release as 2.0.0 when it's done, but maybe consistently using "next release" would be more convenient and allow us to decide on the version number as we cut the release?

Just a thing to consider.

Author: ```suggestion
## [Next Release]
```

> ## [2.0.0] - WIP

Reviewer: ```suggestion
## [2.0.0-alpha.1] - TBD
```
Align with the above. You'll also need to add a link ref at the bottom for the compare:
```
[2.0.0-alpha.1]: https://github.com/kong/kubernetes-ingress-controller/compare/1.2.0...2.0.0-alpha.1
```

Author: :+1: resolved with afcccdc932ea8ce88a033e8270f9bd2fbcebdf34

> ## [Next Release]

Author: ```suggestion
## [2.0.0-alpha.1]
```

> **NOTE**: This release is currently in alpha and the release notes may change significantly until the first beta release is cut.

Author: ```suggestion
**NOTE**: This is a draft of release notes for the next major version to be released and the release notes may change significantly until the first beta release is cut.
```

#### Added

- support for [UDP][kong-udp] via the new `UDPIngress` API.

> - specifying single and multi-namespace configuration for the controller is now available via the controller manager CLI's `--watch-namespace` flag.

Reviewer: ```suggestion
- `--watch-namespace` now supports multiple distinct namespaces versus simply supporting all namespaces (the default) or a single namespace. To watch multiple namespaces, use a comma-separated list (for example, `--watch-namespace "namespaceA,namespaceB"`).
```

[kong-udp]:https://konghq.com/blog/kong-gateway-2-2-released/#UDP-Support

#### Breaking changes

> - running the controller manager via the CLI is now done with a subcommand, instead of `manager <args>` you must call `manager controller <args>` (the arguments thereafter remain the same unless otherwise noted below)

Reviewer 2: We should probably fix that. Is it really the case though? If my memory serves me well, there is no longer that subcommand.

Author: You're right, this is fixed now but I hadn't realized it :+1: 
```suggestion
```
